docs/UserGuide.md

```diff
@@ -716,15 +716,15 @@ The code below is a direct quote from the default data file.
 The parameters in the braces indicate which parameter in the [Parameter descriptions](#parameter-descriptions) the input should adhere to.
 
 
-- `name`: Represents the student's name. If empty, input`""`. (NAME)
-- `phone`: Represents the student's phone number. If empty, input`""`. (PHONE)
-- `email`: Represents the student's email. If empty, input`""`. (EMAIL)
-- `address`: Represents the student's address. If empty, input`""`. (ADDRESS)
-- `remark`: Represents the remark or notes for a student and can contain the newline character `\n`. If empty, input`""`. (REMARK)
-- `education`: Represents the education level of a student. If empty, input`""`. (EDUCATION)
-- `telegram`: Represents the telegram handle of a student. If empty, input`""`. (TELEGRAM)
-- `modules`: Represents an array of modules the student takes. If empty, input`[ ]`. (MODULE)
-- `tagged`: Represents an array of tags the student is labelled with. If empty, input`[ ]`. (TAG)
+- `name`: Represents the student's name. Cannot be empty. (NAME)
+- `phone`: Represents the student's phone number. If empty, delete the entire "phone" row. (PHONE)
+- `email`: Represents the student's email. If empty, delete the entire "email" row. (EMAIL)
+- `address`: Represents the student's address. If empty, delete the entire "address" row. (ADDRESS)
+- `remark`: Represents the remark or notes for a student and can contain the newline character `\n`. If empty, input`""` or delete the entire "remark" row. (REMARK)
```

Comment:
no need to say what i can contain. since we didn't say the constraints of the other fields.

might also wanna tell them to refer to the field constraints thats mentioned in another section.